apps/src/sites/studio/pages/blocks/edit.js

```diff
@@ -37,7 +37,8 @@ function initializeEditPage(defaultSprites) {
   Blockly.inject(document.getElementById('blockly-container'), {
     assetUrl,
     valueTypeTabShapeMap: valueTypeTabShapeMap(Blockly),
-    typeHints: true
+    typeHints: true,
+    isEditMode: true
```

Comment:
Can we call this `isBlockEditMode` instead? When I saw it in the wrapper above first, I wasn't sure quite what the option name meant.

Reply:
yep will update!